docs/getting_started/audioclips.rst

```diff
@@ -19,31 +16,25 @@ AudioClips are very similar to video clips in moviepy: they have a length, can b
 Creating a new audio clip
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Audio clips can be created from an audio file or from the soundtrack of a video file
-::
+Audio clips can be created from an audio file or from the soundtrack of a video file ::
+
     from moviepy.editor import *
     audioclip = AudioFileClip("some_audiofile.mp3")
     audioclip = AudioFileClip("some_video.avi")
 
-Alternatively you can get the audio track of an already created video clip:
+for more, see :py:class:`~moviepy.audio.io.AudioFileClip.AudioFileClip`.
+
+Alternatively you can get the audio track of an already created video clip ::
 
     videoclip = VideoFileClip("some_video.avi")
     audioclip = videoclip.audio
 
-You can also
-
 Compositing audio clips
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-
 Exporting and previewing audio clips
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
`You can also export assign an audio clip as the soundtrack of a video clip with :: `

I'm guessing this should just be "assign"?